### PR TITLE
Config with default template

### PR DESCRIPTION
**Pull request by gforcada**
Drop CMFPlone dependency on the way as well 🎉

### CODE REVIEW

File: plone/app/iterate/configure.zcml [hunk 1]
```diff
@@ -46,20 +46,6 @@
       directory="profiles/default"
       />
 
-  <!--
-    Remove, when CMFPlone no more depends on this profile.
-    Products.CMFPlone/Products/CMFPlone/MigrationTool.py:99:
-    Addon(profile_id=u'plone.app.iterate:plone.app.iterate')
-  -->
-  <genericsetup:registerProfile
-      name="plone.app.iterate"
```
Comment (mauritsvanrees) [inline plone/app/iterate/configure.zcml]: Please keep this profile and the related for now.  We have only implemented the real cleanup in a `plone.app.upgrade` step in Plone 6.0.3.
See https://github.com/plone/plone.app.iterate/issues/99#issuecomment-1484686642

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
Thanks.
Mostly good, but I added two inline comments that should be addressed.

File: plone/app/iterate/configure.zcml (hunk 1 above)
Comment (gforcada) [reply plone/app/iterate/configure.zcml]: Oh, sorry, I thought, although I did not search for when that happened (🤚🏾 lazy me 😓 ).

After having seen a few >2015 old changes that were still lying around I assumed it was yet another one, sorry, will revert 😄 

Though that will bring unfortunately CMFPlone dependency in 😕

File: plone/app/iterate/__init__.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 
 try:
-    import plone.app.relationfield
+    import plone.app.relationfield  # noqa: F401
```
[the remark this answers is not in the document]
Comment (gforcada) [reply plone/app/iterate/__init__.py:33]: that's good to know, thanks! 😄

**Comment by gforcada** [comment]
@jenkins-plone-org

**Review by mauritsvanrees — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
I added two minor commits.

@jenkins-plone-org please run jobs

**Pull request merged**